testgrid/pkg/configurator/prow/prow.go

```diff
@@ -163,6 +163,14 @@ func (pac *ProwAwareConfigurator) ApplySingleProwjobAnnotations(c *configpb.Conf
 		testGroup.DisableProwjobAnalysis = dpaBool
 	}
 
+	if nfta, ok := j.Annotations[testgridNumFailuresToAlertAnnotation]; ok {
+		nftaInt, err := strconv.ParseInt(nfta, 10, 32)
+		if err != nil {
+			return fmt.Errorf("%s value %q is not a valid integer", testgridNumFailuresToAlertAnnotation, nfta)
+		}
+		testGroup.NumFailuresToAlert = int32(nftaInt)
```

Comment:
Since this is a deprecated field, it would be good to add a comment explaining why we're populating it at all. A link to the sourcing bug (https://github.com/GoogleCloudPlatform/testgrid/issues/617) may be useful too.